spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -306,6 +320,56 @@ public RabbitAdmin rabbitAdmin(ConnectionFactory connectionFactory) {
 			return new RabbitAdmin(connectionFactory);
 		}
 
+		@Bean
+		public MultiListenerBean multiListener() {
+			return new MultiListenerBean();
+		}
+
+	}
+
+	@RabbitListener(bindings = @QueueBinding
+			(value = @Queue,
+			exchange = @Exchange(value = "multi.exch", autoDelete = "true"),
+			key = "multi.rk"))
+	static class MultiListenerBean {
+
+		@RabbitListener
```

Comment:
Does this mean that only methods with `@RabbitListener` are candidates for multi-method listener ?